samples/DotNet/Microsoft.Azure.ServiceBus/BasicSendReceiveUsingQueueClient/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BasicSendReceiveUsingQueueClient
+{
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Primitives;
+    using System;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    class Program
+    {
+        const string ServiceBusConnectionString = "{ServiceBus connection string}";
```

Comment:
@SeanFeldman  I feel asking a user to add an environment variable just to use a sample is a bit too much to ask from the user. If we really need to split it out, we could have a config file of some sorts?